test/glific/groups/whatsapp_message_test.exs

```diff
@@ -0,0 +1,87 @@
+defmodule Glific.Groups.WhatsappMessageTest do
+  use Glific.DataCase, async: false
+  use ExUnit.Case
+
+  alias Glific.Providers.Maytapi.Message
+
+  alias Glific.{
+    Partners,
+    Seeds.SeedsDev
+  }
+
+  setup do
+    organization = SeedsDev.seed_organizations()
+
+    Partners.create_credential(%{
+      organization_id: organization.id,
+      shortcode: "maytapi",
+      keys: %{},
+      secrets: %{
+        "phone" => "917834811114",
+        "phone_id" => "42093",
+        "product_id" => "3fa22108-f464-41e5-81d9-d8a298854430",
+        "token" => "f4f38e00-3a50-4892-99ce-a282fe24d041"
+      },
+      is_active: true
+    })
+
+    Tesla.Mock.mock(fn
+      %{
+        method: :post,
+        url: "https://api.maytapi.com/api/3fa22108-f464-41e5-81d9-d8a298854430/42093/sendMessage"
+      } ->
+        %Tesla.Env{
+          status: 200,
+          body: "{\"data\": \"Message sent successfully\"}"
```

Comment:
The actual response structure for `sendMessage` api is 
```json
{"success":true,"data":{"chatId":"78341114@c.us","msgId":"a3ff8460-c710-11ee-a8e7-5fbaaf152c1d"}}
```
shouldn't we use this structure?